opm/polymer/PolymerProperties.cpp

```diff
@@ -243,6 +243,52 @@ namespace Opm
         }
     }
 
+    void PolymerProperties::effectiveInvPolyVis(const double c,
```

Comment:
This function should be named `effectiveInvPolyVisc` (with a c) for consistency.